### PR TITLE
sci-libs/hdf5: version bump to 1.10.5

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-2.3.62, Repoman-2.3.12
Signed-off-by: Horea Christian <horea.christ@yandex.com>

### CODE REVIEW

File: sci-libs/hdf5/hdf5-1.10.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (Zlogene) [inline sci-libs/hdf5/hdf5-1.10.5.ebuild]: do not use obsolete EAPI

File: sci-libs/hdf5/hdf5-1.10.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+FORTRAN_NEEDED=fortran
+AUTOTOOLS_AUTORECONF=1
+
+inherit autotools-utils eutils fortran-2 flag-o-matic toolchain-funcs multilib versionator prefix
```
Comment (Zlogene) [inline sci-libs/hdf5/hdf5-1.10.5.ebuild]: autotools-utils is banned, eutils is not needed, multilib is not needed, versionator should be replaced

File: sci-libs/hdf5/hdf5-1.10.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+MY_P=${PN}-${PV/_p/-patch}
+MAJOR_P=${PN}-$(get_version_component_range 1-2 ${PV})
```
Comment (Zlogene) [inline sci-libs/hdf5/hdf5-1.10.5.ebuild]: and here too

File: sci-libs/hdf5/hdf5-1.10.5.ebuild [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+DESCRIPTION="General purpose library and file format for storing scientific data"
+HOMEPAGE="http://www.hdfgroup.org/HDF5/"
```
Comment (Zlogene) [inline sci-libs/hdf5/hdf5-1.10.5.ebuild]: use https

File: sci-libs/hdf5/hdf5-1.10.5.ebuild [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+SRC_URI="http://www.hdfgroup.org/ftp/HDF5/releases/${MAJOR_P}/${MY_P}/src/${MY_P}.tar.bz2"
```
Comment (Zlogene) [inline sci-libs/hdf5/hdf5-1.10.5.ebuild]: here too

File: sci-libs/hdf5/hdf5-1.10.5.ebuild [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+	threads? ( !cxx !mpi !fortran !hl )"
+
+RDEPEND="
+	mpi? ( virtual/mpi[romio] )
+	szip? ( virtual/szip )
+	zlib? ( sys-libs/zlib:0= )"
+
+DEPEND="${RDEPEND}
+	sys-devel/libtool:2
+	>=sys-devel/autoconf-2.69"
```
Comment (Zlogene) [inline sci-libs/hdf5/hdf5-1.10.5.ebuild]: probably you wanna use `WANT_AUTOCONF`

File: sci-libs/hdf5/hdf5-1.10.5.ebuild (hunk 1 above)
Comment (TheChymera) [reply sci-libs/hdf5/hdf5-1.10.5.ebuild]: The ebuild will break with 6 or 7

File: sci-libs/hdf5/hdf5-1.10.5.ebuild (hunk 2 above)
Comment (TheChymera) [reply sci-libs/hdf5/hdf5-1.10.5.ebuild]: ebuild breaks without `autotools-utils`.

File: sci-libs/hdf5/hdf5-1.10.5.ebuild (hunk 1 above)
Comment (bircoph) [reply sci-libs/hdf5/hdf5-1.10.5.ebuild]: Rewrite it not to break.

File: sci-libs/hdf5/hdf5-1.10.5.ebuild (hunk 2 above)
Comment (bircoph) [reply sci-libs/hdf5/hdf5-1.10.5.ebuild]: Migrate from autotools-utils. Just removing it from inherit is not sufficient of course.

**Comment by luzpaz** [comment]
any progress on this ?

**Comment by heroxbd** [comment]
In tree, closed by 49aa89d2a6c9ebc6c939a985b271e30724e68815

**Pull request closed without merging**